controllers/os-suse-jeos/pkg/generator/templates/script/suse-jeos.template

```diff
@@ -26,6 +26,7 @@ mkdir -p '{{ $unit.DropIns.Path }}'
 
 until zypper -q install -y docker wget socat jq nfs-client; [ $? -ne 7 ]; do sleep 1; done
 systemctl daemon-reload
+ip link set dev eth0 mtu 1460
```

Comment:
do we need it here as well?

Reply:
Yes, we needed.
This template is used when the controller is running in `script` mode.
The other one is for the `cloud-init` mode.
Both should be the same semantically, but different syntactically(one is `bash` script, the other one is `cloud-init`).